frontend/src/components/Carousel.tsx

```diff
@@ -53,7 +53,7 @@ export function Carousel({ slides }: slides) {
             <img
               src={slide.image}
               alt={slide.alt}
-              class="embla__image rounded-lg h-[60vh] bigPhone:h-[70vh] max-w-[85%] object-cover"
```

Comment:
Slight clipping on the bottom: 
![image](https://github.com/wellendorfmatthew/hana-solidjs/assets/157227076/29b7c46d-5a61-43ec-925e-f7a3ff97b9c1)